label_sync/Dockerfile

```diff
@@ -0,0 +1,25 @@
+# Copyright 2017 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM golang
+RUN mkdir src/k8s.io
+WORKDIR /go/src/k8s.io/
+RUN git clone https://github.com/kubernetes/test-infra
+WORKDIR /go/src/k8s.io/test-infra
+ADD ./label_sync/ /go/src/k8s.io/test-infra/label_sync/
+ADD ./labels.yaml /etc/config/labels.yaml
```

Comment:
Not sure how to do that, but I will have a discussion on Hangouts today 10:30 am PDT. I'm syncing `labels.yaml` and then I'll try to create minimal docker image/Dockerfile. Then I will create YAML with that image connected and also with secrets connected.
But this is my first time writing some Kubernetes code, so I have no idea how to do it actually.